pkg/auth/iam/iam.go

```diff
@@ -63,14 +63,18 @@ func (filter *Filter) Auth(opts ...FilterOption) restful.FilterFunction {
 		token, err := parseAccessToken(req)
 		if err != nil {
 			logrus.Warn("unauthorized access: ", err)
-			logErr(resp.WriteErrorString(http.StatusUnauthorized, "unauthorized access"))
+			errorRespond := respondError(http.StatusUnauthorized, ErrorCodeUnauthorizedAccess,
+				"unauthorized access")
+			logErr(resp.WriteErrorString(errorRespond.Code, errorRespond.Message))
 			return
 		}
 
 		claims, err := filter.iamClient.ValidateAndParseClaims(token)
 		if err != nil {
 			logrus.Warn("unauthorized access: ", err)
-			logErr(resp.WriteErrorString(http.StatusUnauthorized, "unauthorized access"))
+			errorRespond := respondError(http.StatusUnauthorized, ErrorCodeUnauthorizedAccess,
```

Comment:
there is still hardcoded response body in the respondError fnction

Reply:
do you mean the "unauhorized access" error message is the hard coded one? If it is, I will update it. Thanks.

Reply:
try to open the function def of respondError( )

Reply:
I have checked the function and it looks like respondError() function returns serviceError and it has message field that needs string. If we update the hardcoded response body, we need to marshal the errorResponse. If we need to change this, is it okay if I don't check the json.marshal of the new errorResponse?